common/lib/xmodule/xmodule/templates/problem/jsinput_response.yaml

```diff
@@ -5,67 +5,176 @@ metadata:
+          
+          if farthest &lt; tolerance[0]:
+            final_grade = brackets[0]
+            message = 'Close enough! Actual answer: ' + str(correct_number)
+          elif farthest &lt; tolerance[1]:
+            final_grade = brackets[1]
+            message = 'Close. You are off by ' + str(farthest)
+          elif farthest &lt; tolerance[2]:
+            final_grade = brackets[2]
+            message = 'Not very close. You are off by ' + str(farthest)
+          else:
+            final_grade = brackets[3]
+            message = 'Your range is too large to get points.'
+          
+          if guess_upper &gt; correct_number and guess_lower &lt; correct_number:
+            message += ' The answer is within your range.'
+          else:
+            message += ' The answer is outside your range.'
+          
+        if not feedback:
+          message = ''
+
+        if final_grade &gt; 0.95:
+          isOK = True
+        elif final_grade &gt; 0.05:
+          isOK = "Partial"
+        else:
+          isOK = False
+        
+        return {
+          'input_list': [
+              { 'ok': isOK, 'msg': message, 'grade_decimal': final_grade},
+          ]
+        }
+
+
+
+      </script>
+        <script type="text/javascript">
+      $(document).ready(function(){
+      	console.log('Let\'s go!');
+      });
+
+      // Logs a javascript object.
+      function logThatThing(ThatThing){
+
+      	// Log it to the console just to verify it's working
+      	console.log(JSON.stringify(ThatThing));
+      	
+      	// Send it to the official edX logamajig!
+      	// Logger.log("harvardx.public_demo.range_guesser", ThatThing);
+
+      }
       </script>
-            <p>In the following image, click the objects until the cone is yellow and the cube is blue.</p>
-            <jsinput gradefn="WebGLDemo.getGrade"
-              get_statefn="WebGLDemo.getState"
-              set_statefn="WebGLDemo.setState"
-              initial_state='{"selectedObjects":{"cube":true,"cylinder":false}}'
-              width="400"
-              height="400"
-              html_file="https://studio.edx.org/c4x/edX/DemoX/asset/webGLDemo.html"
-              sop="false"/>
-          </customresponse>
+        <p>What is the range of passengers that can fit on a 747 aircraft? Set the low and high bounds. (416 and 660 persons)</p>
+        <p class="sr" aria-hidden="true">
+          <span id="lowerlimit">$lowerlimit</span>
+          <span id="upperlimit">$upperlimit</span>
+          <span id="openclose">$show_open_close</span>
+          <span id="istimequestion">$is_time_question</span>
+          <span id="maxtime">$max_time</span>
+        </p>
+        <customresponse cfn="answercheck">
+          <jsinput gradefn="guesser.getGrade" get_statefn="guesser.getState" set_statefn="guesser.setState" width="800" height="120" html_file="/static/guesser.html" sop="false"/>
```

Comment:
It's one of the files that came with this example from Harvard. It's on files.edx.org/custom-js/